scss/_alert.scss

```diff
@@ -3,13 +3,24 @@
 //
 
 .alert {
+  // scss-docs-start alert-css-vars
+  --#{$variable-prefix}alert-bg: transparent;
+  --#{$variable-prefix}alert-padding: #{$alert-padding-y $alert-padding-x};
+  --#{$variable-prefix}alert-margin-bottom: #{$alert-margin-bottom};
+  --#{$variable-prefix}alert-color: #{$alert-padding-y $alert-padding-x};
+  --#{$variable-prefix}alert-border-color: transparent;
```

Comment:
Shouldn't we use this on l.69 in this file ? Instead of ```border-color: $value```, having ```--#{$variable-prefix}alert-border-color: $value``` should be more flexible. However, I'm not sure if the flexible argument is a real one here.

Reply:
Not sure to understand what you propose here :innocent: